lambda/executor.js

```diff
@@ -15,6 +15,10 @@ module.exports.handler = async(event, context) => {
 
     validateInput(lambdaARN, value, num); // may throw
 
+    // get minCost map and min cost
+    const minCostsMap = generateBasePriceMap(minCosts, lambdaARN)
```

Comment:
I guess you don't need to pass `lambdaARN` as second argument here? Since it doesn't depend on the input event, this env variable parsing could happen out of the handler?

Reply:
The implementation needs to know in which region this lambda is running to return the appropriate cost.

Reply:
But `generateBasePriceMap` only has one input parameter :)

Reply:
Hahahaha.
My bad.
Rewrote this a bit please check.